### PR TITLE
Bugfix/#17140 remove chef dep

### PR DESCRIPTION
**Pull request by ljblancoredborder**
Chef dependency where being generated because just a an info message, which caused the bug. We should try to reduce dependecies as much as possible.

### CODE REVIEW

File: resources/scripts/rb_check_cgroups.rb [hunk 1]
```diff
@@ -2,23 +2,21 @@
 # frozen_string_literal: true
 
 require 'socket'
```
Comment (malvads) [inline resources/scripts/rb_check_cgroups.rb]: Why socket dependency here?

**Review by malvads — CHANGES_REQUESTED** [review]

File: resources/scripts/rb_check_cgroups.rb (hunk 1 above)
Comment (ljblancoredborder) [reply resources/scripts/rb_check_cgroups.rb]: Appendix to remove :)

**Review by malvads — APPROVED** [review]
Now looks good to me

**Comment by malvads** [comment]
remember to change base branch

**Pull request merged**